VTCareService/src/main/java/edu/vt/cs/vtcare/vtcareservice/api/VTCareResource.java

```diff
@@ -20,4 +23,30 @@ public Provider createProvider(Provider provider) throws Exception {
             throw new Exception("An exception occurred while persisting the provider details.");
         }
     }
+
+    @POST
+    @Path("login")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
+    public Response login(String json) throws Exception {
```

Comment:
(String json) : json is not a very informative and readable name for the parameter. It should be renamed to something that give more idea about what it contains.

Reply:
Avoid making generic variable names like json. It is very confusing.